code/modules/surgery/organs/subtypes/machine.dm

```diff
@@ -115,15 +115,22 @@
 
 	. = ..()
 
+/obj/item/organ/internal/brain/mmi_holder/posibrain
+	name = "positronic brain"
+
 /obj/item/organ/internal/brain/mmi_holder/posibrain/New()
 	..()
 	stored_mmi = new /obj/item/mmi/robotic_brain/positronic(src)
-	if(owner)
-		stored_mmi.name = "positronic brain ([owner.real_name])"
-		stored_mmi.brainmob.real_name = owner.real_name
+	if(!owner)
+		stored_mmi.forceMove(get_turf(src))
+		qdel(src)
+
+/obj/item/organ/internal/brain/mmi_holder/posibrain/remove(mob/living/user, special = 0)
+	if(stored_mmi && dna)
+		stored_mmi.name = "[initial(name)] ([dna.real_name])"
```

Comment:
This *does* technically mean that posibrains/robotic brains are just "posibrain" when inside an IPC now (you can't see the specific name during surgery), but, quite frankly, this is a good thing and is consistent with how brains are; that said, of course, when actually extracted, they will have the proper name.